### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by MRDGH2821**
Added Language badges
Added MIT badge
Resolved common English errors. 

Badge credits - 
https://github.com/ileriayo/markdown-badges/
https://shields.io/

### CODE REVIEW

**Comment by MRDGH2821** [comment]
You can make a new branch, merge the PR there & review the changes. And finally into the main branch

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -4,12 +4,13 @@
 <h4 align="center"> Developed by: Kaustubh Gupta </h4>
 <img src="./images/preview.PNG" alt="Screeenshot taken on 11th October, 2020" align="center">
 </div>
+![Python](https://img.shields.io/badge/python%20-%2314354C.svg?&style=for-the-badge&logo=python&logoColor=white) ![Docker](https://img.shields.io/badge/docker%20-%230db7ed.svg?&style=for-the-badge&logo=docker&logoColor=white) ![Vercel](https://img.shields.io/badge/vercel%20-%23000000.svg?&style=for-the-badge&logo=vercel&logoColor=white) ![Markdown](https://img.shields.io/badge/markdown-%23000000.svg?&style=for-the-badge&logo=markdown&logoColor=white)
```
[the remark this answers is not in the document]
Comment (MRDGH2821) [reply README.md]: Changes done.
New commit - 036e7f4

Removed Vercel badge.
Fixed Rendering of other badges.

Somehow, those badges get rendered only when there is empty line above & below them🤔

**Comment by MRDGH2821** [comment]
Changes done.
New commit - 036e7f4

Removed Vercel badge.
Fixed Rendering of other badges.

Somehow, those badges get rendered only when there is empty line above & below them🤔

Might need to use HTML tags instead of Markdown syntax. Which I'm not familiar with 😅

**Review by kaustubhgupta — APPROVED** [review]

**Comment by kaustubhgupta** [comment]
Thanks for contributing!

**Comment by MRDGH2821** [comment]
Welcome! ☺️

**Comment by kaustubhgupta** [comment]
That's great!

**Pull request merged**